src/app/storymaps/tpl/core/Maptiks.js

```diff
@@ -0,0 +1,21 @@
+define(["dojo/topic"],
+  function(topic){
+    topic.subscribe("story-loaded-map", function(){
+      require(['maptiks'], function (mapWrapper) {
+        if (!app.map.maptiks) {
+          var container = app.map.container; // the current map div
+          var maptiksSet = app.data.getWebAppData().getMaptiks().hasOwnProperty('maptiksTrackcode');
+          var maptiksTrackcode = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksTrackcode : 'c311cf16-ad79-42b1-97f9-f433be6c8b00'; // replace with dev trackcode
+          var maptiksId = maptiksSet ? app.data.getWebAppData().getMaptiks().maptiksId : "demo";
```

Comment:
I don't want to see these in the code that all our templates will use. Can this belong on the demo.html page?